src/Components/Header/Header.js

```diff
@@ -1,7 +1,76 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import MenuDropDown from './MenuDropDown';
+import ProfileModal from './ProfileModal';
+import SearchForm from '../../Pages/Main/SearchForm';
+
+import './Header.scss';
+import ModalPortal from '../../Portal';
 
 const Header = () => {
-  return <div />;
+  const url = document.location.href;
+
+  const [user, setUser] = useState({});
+
+  const [menuOpened, setMenuOpened] = useState(false);
+
+  const [profileClicked, setProfileClicked] = useState(false);
+
+  const profileClick = () => {
+    setProfileClicked(true);
+  };
+
+  const menuOpen = () => {
+    setMenuOpened(true);
+  };
+
+  const menuClose = () => {
+    setMenuOpened(false);
+  };
+
+  useEffect(() => {
+    fetch('./jiwonData/loginUser.json')
+      .then(res => res.json())
+      .then(data => setUser(data));
+  }, []);
+
+  const navigate = useNavigate();
+
+  return (
+    <div className="header">
+      <div className="headerRight" onClick={() => navigate('/')}>
+        <img className="headerLogoImg" src={LOGO_IMAGE} alt="로고이미지" />
+        <img className="headerLogoImg" src={LOGO_TEXT_IMAGE} alt="로고이미지" />
+      </div>
+
+      {url !== 'http://localhost:3000/' && <SearchForm />}
+
+      <div className="headerLeft">
+        <img
+          className="headerCategories"
+          src={CATEGORY_IMAGE}
+          alt="메뉴 열기"
+          onMouseOver={menuOpen}
+        />
+        <img
+          className="headerProfileImg"
+          src={user.user_profile_img}
+          alt="프로필 이미지"
+          onClick={profileClick}
```

Comment:
```suggestion
          onClick={()=>setProfileClicked(true)}
```
해당 함수가 단순 state 변경 로직만 있다면 위와 같은 방법으로 구현해주세요!